backend/src/test/java/com/redhat/cloud/notifications/db/DbCleaner.java

```diff
@@ -59,42 +58,32 @@
     /**
      * Deletes all records from all database tables (except for flyway_schema_history) and restores the default records.
      * This method should be called from a method annotated with <b>both</b> {@link BeforeEach} and {@link AfterEach} in
-     * all test classes that involve SQL queries to guarantee the tests isolation in terms of stored data. Ideally, we
-     * should do that with {@link io.quarkus.test.TestTransaction} but it doesn't work with Hibernate Reactive, so this
```

Comment:
Unfortunately, `@TestTransaction` does not work with RestAssured requests, so we'll keep `DbCleaner` permanently.

Reply:
At least we know that is permanent :D